### ISSUE TITLE
Remove `default_workitem_type` option 

### ISSUE THREAD

**Issue by timvink** [ing-bank/doing-cli#66]
Currently, we have an optional `default_workitem_type` setting in the config that controls the work item type when using `doing create issue` and `doing workon`.

Instead, we should encourage & document the use of `defaults`:

```yaml
# .doing-cli-config.yml

defaults:
    DOING_CREATE_ISSUE_TYPE: 'Bug'
    DOING_WORKON_TYPE: 'Bug'
```

**Comment by timvink** [comment]
Because this is a breaking change, we can build in a deprecation error in `get_config()` when that parameter is called.

**Comment by timvink** [comment]
Released in [v1.2.2](https://github.com/ing-bank/doing-cli/releases/tag/v1.2.2)
